src/content/self-hosted/install/releases.mdx

```diff
@@ -6,11 +6,29 @@ id: releases
 ---
 
 ## 1.11.0
-TBD
+2 August 2023
+
+### Before you upgrade
+
+Okteto 1.11 enables by default ingress-nginx validation webhook (adding an additional Kubernetes deployment), which protects the platform against single user ingress syntax errors. <!-- 6706 -->
```

Comment:
```suggestion
In Okteto 1.11 the ingress-nginx validation webhook (adding an additional Kubernetes deployment) is enabled by default. This webhook validates any new ingress objects that are created or updated throughout the cluster to protect the platform from single user ingress syntax errors. <!-- 6706 -->
```

Non-blocking suggestion. Goal is to over-explain and not assume what our users do/don't know about Kubernetes or Okteto

Can someone expand more on